data/json/items/ranged.json

```diff
@@ -4213,6 +4213,38 @@
     "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
   },
   {
+    "id": "m41a_pulse_rifle",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "teal",
+    "name": "M41A pulse rifle",
+    "description": "A powerful electrolaser  developed by the R&D outfit \"Aerial Industries\".  Especially effective against electronic targets.",
+    "price": 1600000,
+    "material": ["steel", "plastic"],
+    "ammo_effects": ["DRAW_LASER_BEAM", "EMP", "BEANBAG"],
```

Comment:
That sounds more like a sign that the beanbag needs to be nerfed. We have a stronger version of the flag anyway, right?

It would make more sense if we had an ammo flag that stunned based of electrical resistance, so shockers could presumably no-sell it.